scripts/start_civ.sh

```diff
@@ -514,6 +526,10 @@ function parse_arg() {
                 set_pt_usb
                 ;;
 
+            --passthrough-pci-udc)
```

Comment:
Add description in show_help() function.

Reply:
Sure.